zuul.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- project:
+    templates:
+      - ansible-navigator-jobs
```

Comment:
now that this is passing, lets remove this file. and add project-config / ansible-zuul-jobs changes.